spec/msgr/msgr/dispatcher_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+class MsgrAutoAckConsumer < Msgr::Consumer
+  auto_ack true
```

Comment:
This is the default, right? What about adding a method like `require_(manual)_ack` instead that only needs to be called to change the default. Might be better than a boolean parameter.

Reply:
Hmm, I actually thought about making it an actual attribute writer such as `self.auto_ack = false`.